src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final Color color;
```

Comment:
Color property is included into parent class,  no need to duplicate it in children classes.
Please recheck all the figures for this